redis-lettuce/src/main/java/io/micronaut/configuration/lettuce/AbstractRedisConfiguration.java

```diff
@@ -80,6 +83,22 @@ public void setUris(URI... uris) {
         this.uris = Arrays.stream(uris).map(RedisURI::create).collect(Collectors.toList());
     }
 
+    /**
+     * @return Get the Redis URIs for read replicas.
+     */
+    public List<RedisURI> getReplicaUris() {
```

Comment:
could you add `@since 6.5.0` to all new pubic methods.

Reply:
please declare nullability annotations on all new methods and arguments (Micronaut's `Nullable` and `@NonNull`. If anything can be `null` consider returning `Optional`